docs-source/content/userguide/managing-domains/model-in-image/usage.md

```diff
@@ -32,22 +32,30 @@ First, obtain a base image:
 | `/u01/wdt/models`         | Application archives               | `.zip`        |
-| `/u01/wdt/weblogic-deploy`| Unzipped WebLogic deploy install   |             |
+| `/u01/wdt/weblogic-deploy`| Unzipped WDT install binaries     |             |
 
-> **Note**: Model YAML and variable files are optional in a Model in Image image `/u01/wdt/models` directory because Model in Image also supports [supplying them dynamically]({{< relref "/userguide/managing-domains/model-in-image/runtime-updates.md" >}}) using a ConfigMap referenced by the Domain YAML file `spec.model.configMap` field. Application archives, if any, must be supplied in the Model in Image image.  Application archives are not supported in a `spec.model.configMap`.
+Notes:
 
-There are two methods for layering Model in Image artifacts on top of a base image:
+  - The `/u01/wdt/models` directory is the Model in Image _model home_; for advanced use cases, you can customize its location using the Domain YAML file `configuration.model.modelHome` field.
+
+  - Model YAML and variable files are optional in a Model in Image image model home directory because Model in Image also supports [supplying them dynamically]({{< relref "/userguide/managing-domains/model-in-image/runtime-updates.md" >}}) using a ConfigMap referenced by the Domain YAML file `spec.model.configMap` field.
+
+  - Application archives, if any, must be supplied in the model home. Application archives are not supported in a `spec.model.configMap`.
+
+There are three methods for layering Model in Image artifacts on top of a base image:
 
   - Manual Image Creation: Use Docker commands to layer the WDT artifacts from the above table on top of your base image into a new image.
 
-  - WebLogic Image Tool: Use the [WebLogic Image Tool](https://github.com/oracle/weblogic-image-tool). The WebLogic Image Tool (WIT) has built-in options for embedding WDT model files, WDT binaries, WebLogic Server binaries, and WebLogic Server patches in an image. The [Model in Image]({{< relref "/samples/simple/domains/model-in-image/_index.md" >}}) sample uses the WIT approach.
+  - WebLogic Image Tool: Use the [WebLogic Image Tool](https://github.com/oracle/weblogic-image-tool). The WebLogic Image Tool (WIT) has built-in options for layering WDT model files, WDT binaries, WebLogic Server binaries, and WebLogic Server patches in an image. The [Model in Image]({{< relref "/samples/simple/domains/model-in-image/_index.md" >}}) sample uses the WIT approach.
+
+  - Persistent Volume Claim: This method is for advanced use cases only. Supply WDT model YAML, variable, or archive files in a [Persistent Volume Claim]({{< relref "/faq/volumes.md" >}}) and modify `configuration.model.modelHome` to the corresponding directory within the PVC's mount location. This method does not address the requirement to include an unzipped WDT install binaries in the `/u01/wdt/weblogic-deploy` directory (use either of the previous two methods).
```

Comment:
to include an unzipped WDT install binaries -> to include unzipped WDT install binaries

Reply:
Fixed.